### PR TITLE
Update feed.ts to handle LIVE_CHAT_MESSAGE (1311) correctly:

### LINKED ISSUE #380: Show comments in Coracle
Zapstream and nests use kind 1311. Add them to noteKinds, and in feed.ts only show them if there's a parent (in `buildFeedChunk`).

### PR DESCRIPTION
**Pull request by LeParrain0**
Resolves #380 (second attempt but i really hope it works this time)
So, i
- Replaced numeric literal 1311 with imported LIVE_CHAT_MESSAGE constant.
- Adjusted buildFeedChunk to handle parent events and filtering.
- Added filtering for LIVE_CHAT_MESSAGE to ensure it only appears if it has a parent.

And please this time explain how i can manually test it to ensure it works ! I am really happy to contribute.

### CODE REVIEW

File: src/app/util/feeds.ts [hunk 1]
```diff
@@ -48,7 +49,7 @@ export type FeedOpts = {
 
 const prepFilters = (filters, opts: FeedOpts) => {
   // Default to note kinds
-  filters = filters?.map(filter => ({kinds: noteKinds, ...filter})) || []
+  filters = filters?.map(filter => ({kinds: [...noteKinds, LIVE_CHAT_MESSAGE], ...filter})) || []
```
Comment (staab) [inline src/app/util/feeds.ts:52]:
```suggestion
  filters = filters?.map(filter => ({kinds: noteKinds, ...filter})) || []
```

File: src/app/util/feeds.ts [hunk 2]
```diff
@@ -260,6 +261,10 @@ export const createFeed = (opts: FeedOpts) => {
             if (seen.has(getIdOrAddress(e))) return false
             if (repostKinds.includes(e.kind)) return false
             if (reactionKinds.includes(e.kind)) return false
+            if (e.kind === LIVE_CHAT_MESSAGE) {
+              const parentIds = Tags.fromEvent(e).parents().values().valueOf()
+              return parentIds.length > 0 // Ne montrer que s'il y a un parent
+            }
```
Comment (staab) [inline src/app/util/feeds.ts:267]:
```suggestion
            if (e.kind === LIVE_CHAT_MESSAGE && !Tags.fromEvent(e).parents().exists()) return false
```

**Review by staab — APPROVED** [review]

**Comment by staab** [comment]
Thanks for the contribution!

**Comment by LeParrain0** [comment]
> Thanks for the contribution!

You are welcome sir; it's a pleasure to work on your project, but i won't say no to a tip ^^

**Pull request merged**